.aicoe-ci.yaml

```diff
@@ -5,9 +5,10 @@ check:
   - thoth-build
 
 build:
-  build-stratergy: Source
+  build-strategy: Source
```

Comment:
:( Unfortunately, there is a typo of `build-stratergy` attribute on the aicoe-ci application level.  
`build-strategy` won't work. 
The would-be opened as an issue on aicoe-ci and we would fix it in a gradual manner, so other users don't face issue.